shims/cdpdc71/driver/pom.xml

```diff
@@ -647,6 +642,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hbase</groupId>
```

Comment:
does this have to be applied to all the other relevant shims that will be supported by Pentaho in 10.3?

Reply:
No, this is just repositioning the jar, others already properly placed